bin/cli.js

```diff
@@ -1,16 +1,39 @@
 #!/usr/bin/env node
 /* eslint no-console:0, no-var:0 */
 var Liftoff = require('liftoff');
-var Promise = require('bluebird');
 var interpret = require('interpret');
 var path = require('path');
 var chalk = require('chalk');
 var tildify = require('tildify');
 var commander = require('commander');
 var argv = require('minimist')(process.argv.slice(2));
-var fs = Promise.promisifyAll(require('fs'));
+var fs = require('fs');
 var cliPkg = require('../package');
 
+function readFile(path) {
+  return new Promise(function(resolve, reject) {
```

Comment:
why not just use https://www.npmjs.com/package/es6-promisify?

Reply:
That's an option. I was hoping to avoid adding dependencies, but that package has none of its own so the footprint is small.

Reply:
Promisifying everything by hand is very noisy. Alternatively we could copy promisification code into some internal util and use that to avoid external dependencies - and remove it after we drop Node 6 support.
@elhigu What do you think?

Reply:
But if it's left as-is, more work would be needed to start using native promisification, as every single block that is like this would have to be rewritten. If es6-promisify is used, change is as trivial as switching the import.

Reply:
That's other discussion when bluebird promises should be removed completely and how. 

Yeah I'm not promoting making this kind of promisification everywhere if there are many places where it is needed and if complete codebase is converted to not use bluebird (even though I'm not sure if there even is many places where that has been used in codebase).

Reply:
There's only one more instance of `promisifyAll`, but it looks like 13 of `promisify`. I'm not averse to using a library given that number.

Reply:
> That's other discussion when bluebird promises should be removed completely and how.

The module changed in this PR is a special case. Most others will have to retain bluebird for now, since code which consumes knex may assume that promises returned are bluebird ones.

I suggest first factoring out all use of methods which are not in common with vanilla promises, and then both removing bluebird and allowing a custom promise to be parsed in (see #1588). This'll mean a major version bump, but will provide an upgrade path for those who are bound to bluebird.